dune-project

```diff
@@ -1,3 +1,3 @@
-(lang dune 1.4)
+(lang dune 1.5)
```

Comment:
Why this change?

Reply:
The change in `test-suite/Makefile` requires 1.5.